src/NHSD.BuyingCatalogue.Ordering.Api/Startup.cs

```diff
@@ -65,10 +71,33 @@ public void ConfigureServices(IServiceCollection services)
             });
         }
 
+        public void Configure(IApplicationBuilder app)
+        {
+            var pathBase = Configuration.GetValue<string>("pathBase");
```

Comment:
We currently don't have a requirement for the page base setting within this API. It was only added out of necessity to the Identity.Api project to provide routing from the hosted application within the Kubernetes cluster. This should look more like the Organisations.Api Startup class / Configure method. For clarity reasons I think the use of the pathBase setting could be removed.